README.md

```diff
@@ -439,6 +439,8 @@ __fixedHeader__ - (boolean) fix table header while scrolling. Default is `true`.
 
 __hidePagination__ - Hide paging and search bar if data size is less than pagination limit - Default is `true`
 
+__additionalSearchOperator__ - (boolean) To enable/disable addition search operator for number type column value - Default is `false`
```

Comment:
As per @AnamikaDubey07 suggestion [here](https://github.com/nuagenetworks/vis-graphs/issues/446#issuecomment-780490971), can you enable this option by default since it is required for all the explorer tables? We can disable it when required from the config files.

Reply:
rename the parameter as `enableNumericSearch`. for now, let the default be false